cloud/blockstore/libs/daemon/ydb/bootstrap.cpp

```diff
@@ -251,6 +252,10 @@ void TBootstrapYdb::InitKikimrService()
         .Settings = std::move(settings)
     };
 
+    if (emergencyMode) {
+        registerOpts.SchemeShardDir = "";
```

Comment:
Otherwise we receive an error from there https://github.com/ydb-platform/nbs/blame/214e05a54b76ccf6b201b9836f147540df6d5abf/contrib/ydb/core/mind/node_broker__register_node.cpp#L60